apps/svelte.dev/src/lib/server/content.ts

```diff
@@ -124,5 +126,198 @@ function create_docs() {
 }
 
 export const docs = create_docs();
-
 export const examples = index.examples.children;
+
+export const packages = Array.from(
+	new Set(
+		Object.keys(docs.topics)
+			.map((topic) => topic.split('/')[1])
+			.filter(Boolean)
+	)
+);
```

Comment:
this seems very elaborate — why not just this?

```suggestion
export const packages = Object.keys(docs.topics).map((topic) => topic.split('/')[1]);
```

Reply:
Updated